=====2019/7/30 8:41:32=====
=====2019/7/30 8:41:42=====
DMT02----Error----连接成功192.168.5.102:52691----2019/7/30 8:42:49
DMT02----Error----System.Net.Sockets.SocketException (0x80004005): 远程主机强迫关闭了一个现有的连接。
   在 System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   在 TcpServerRoot.TcpClient.ReceiveCallBack(IAsyncResult ar) 位置 D:\RemoteControl\TcpServerRoot\TcpClient.cs:行号 83----2019/7/30 8:43:21
DMT02----Error----断开连接----2019/7/30 8:43:21
=====2019/7/30 8:48:24=====
=====2019/7/30 8:53:17=====
DMT02----Error----连接成功127.0.0.1:53490----2019/7/30 8:53:44
DMT02----Error----System.Net.Sockets.SocketException (0x80004005): 远程主机强迫关闭了一个现有的连接。
   在 System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   在 TcpServerRoot.TcpClient.ReceiveCallBack(IAsyncResult ar) 位置 D:\RemoteControl\TcpServerRoot\TcpClient.cs:行号 83----2019/7/30 8:54:04
DMT02----Error----断开连接----2019/7/30 8:54:04
DMT02----Error----连接成功127.0.0.1:53664----2019/7/30 8:55:15
DMT02----Error----断开连接----2019/7/30 8:55:39
DMT02----Error----连接成功127.0.0.1:53679----2019/7/30 8:56:11
DMT02----Error----System.Net.Sockets.SocketException (0x80004005): 远程主机强迫关闭了一个现有的连接。
   在 System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   在 TcpServerRoot.TcpClient.ReceiveCallBack(IAsyncResult ar) 位置 D:\RemoteControl\TcpServerRoot\TcpClient.cs:行号 83----2019/7/30 8:56:28
DMT02----Error----断开连接----2019/7/30 8:56:28
=====2019/7/30 17:31:38=====
DMT02----Info----连接成功127.0.0.1:65470----2019/7/30 17:31:39
DMT02----Info----断开连接----2019/7/30 17:31:50
=====2019/7/30 17:33:01=====
DMT02----Info----连接成功127.0.0.1:65482----2019/7/30 17:33:08
